core/services/ardupilot_manager/mavlink_proxy/Endpoint.py

```diff
@@ -1,15 +1,15 @@
 from dataclasses import dataclass
-from enum import IntEnum
+from enum import Enum
 from typing import Dict, Optional, Union
 
 
-class EndpointType(IntEnum):
-    Undefined = 0
-    UDPServer = 1
-    UDPClient = 2
-    TCP = 3
-    Serial = 4
-    File = 5
+class EndpointType(str, Enum):
+    Undefined = "undefined"
+    UDPServer = "udpin"
+    UDPClient = "udpout"
+    TCP = "tcp"
+    Serial = "serial"
+    File = "file"
```

Comment:
Will this break the settings system when it loads with the old settings ?

Reply:
Yes